src/main/java/pack/controller/PersonController.java

```diff
@@ -1,102 +1,88 @@
-package pack.controller.impl;
+package pack.controller;
 
-import java.io.IOException;
-import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
-
 import javax.annotation.PostConstruct;
-
+import javax.ws.rs.core.MediaType;
 import org.apache.http.HttpResponse;
 import org.apache.http.client.HttpClient;
 import org.apache.http.client.methods.HttpGet;
 import org.apache.http.impl.client.HttpClientBuilder;
 import org.apache.http.util.EntityUtils;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.ResponseBody;
-
+import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import pack.controller.IPersonController;
-import pack.model.Address;
-import pack.model.Company;
-import pack.model.Geo;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
 import pack.model.Person;
-import pack.service.IAddressService;
 import pack.service.IPersonService;
+import pack.util.CustomException;
 
 @Controller
-public class PersonController implements IPersonController {
+public class PersonController {
 
+	final JsonNodeFactory factory = JsonNodeFactory.instance;
+	ObjectNode node;
+	ObjectNode child;
 	private HttpClient httpClient;
 	private HttpGet request;
 	private HttpResponse response;
 	private ObjectMapper objectMapper;
-	private List<Person> personsList;
 
 	@Autowired
 	private IPersonService personService;
-	
 
 	@PostConstruct
 	public void init() {
 		httpClient = HttpClientBuilder.create().build();
 		request = new HttpGet("https://jsonplaceholder.typicode.com/users");
 		objectMapper = new ObjectMapper();
-		personsList=new ArrayList<>();
+		node = factory.objectNode();
+		child = factory.objectNode();
 	}
 
-	@RequestMapping(value = "/getPersons", method = RequestMethod.GET)
-	@Override
+	@RequestMapping(value = "/getPersons", method = RequestMethod.GET, produces = "application/json")
 	@ResponseBody
-	public List<Person> getPersons() {
-		return personService.getPersons();
+	public ObjectNode getPersons() throws CustomException, JsonProcessingException {
+
+		try {
+			String json = objectMapper.writerWithDefaultPrettyPrinter().writeValueAsString(personService.getPersons());
+			child.put("status", HttpStatus.ACCEPTED.getReasonPhrase());
```

Comment:
Please return your custom data object with these fields status, data, reason for all endpoints.